File: example-crud-item-list/src/main/java/com/gurunars/crud_item_list/example/EmptyBinder.java
package com.gurunars.crud_item_list.example;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;

import com.gurunars.item_list.EmptyViewBinder;

class EmptyBinder implements EmptyViewBinder {

    @Override
    public View getView(Context context) {
        return LayoutInflater.from(context).inflate(R.layout.completely_empty_list, null);
    }

}

File: examples/crud-item-list/src/main/kotlin/com/gurunars/crud_item_list/example/EmptyBinder.kt
package com.gurunars.crud_item_list.example

import android.content.Context
import android.view.LayoutInflater
import android.view.View

import com.gurunars.item_list.EmptyViewBinder

internal class EmptyBinder : EmptyViewBinder {

    override fun getView(context: Context): View {
        return LayoutInflater.from(context).inflate(R.layout.completely_empty_list, null)
    }

}
